tests/lib/rules/destructuring-assignment.js

```diff
@@ -882,10 +882,16 @@ ${'            '}
         };
       `,
       options: ['always', { destructureInSignature: 'always' }],
-      features: ['types'],
+      features: ['types', 'no-babel'],
```

Comment:
we should still be testing the pre-existing test cases against the babel parser.

Reply:
I changed the destination branch from issue-3828 to master.